UnitySDK/Assets/ML-Agents/Examples/Bouncer/Scripts/BouncerAcademy.cs

```diff
@@ -7,6 +7,7 @@ public class BouncerAcademy : Academy
 
     public override void InitializeAcademy()
     {
+        FloatProperties.RegisterCallback("target_scale", f => { });
```

Comment:
These empty callbacks seem weird. C# can still receive the reset parameters from python without the property being registered, right?

Reply:
Yes it can. I will remove those.